<ui version="4.0" stdsetdef="1">
  <author>Peter Bartfai</author>
  <class>ExportOptionPanel</class>
  <widget class="QDialog" name="ExportOptionPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>362</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>POV Export Options</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <widget class="QScrollArea" name="scrollArea" >
          <property name="verticalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy" >
            <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" name="buttonsLayout">
          <item>
            <widget class="QPushButton" name="resetButton">
              <property name="text">
                <string>Reset</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer1">
              <property name="sizeHint">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="text">
                <string>Ok</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="text">
                <string>Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
